samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -183,6 +183,17 @@
     "LongTermMemoryName": "LongTermMemory",
     "LongTermMemoryExtraction": "Extract information that is encoded and consolidated from other memory types, such as working memory or sensory memory. It should be useful for maintaining and recalling one's personal identity, history, and knowledge over time."
   },
+  //
+  // Content Moderation. Currently only supports Azure Content Moderator. 
+  // - Set "Endpoint" to the endpoint of your Azure Content Moderator instance (e.g., "https://contoso.openai.azure.com")
+  // - Set "Key" to the endpoint of your Azure Content Moderator instance using dotnet's user secrets 
+  //       (i.e. dotnet user-secrets set "ContentModeration:Key" "MY_API_KEY")
+  "ContentModeration": {
+    "Enabled": true,
```

Comment:
Do we want that enabled by default? Left as is, this will crap out since we don't have an endpoint and key set.

If set to false, CC will just work fine, just without content moderation.

Reply:
Changed it false by default